src/lib/statistics/cardinality_estimation_cache.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "join_graph_statistics_cache.hpp"
+
+namespace opossum {
+
+class CardinalityEstimationCache {
+ public:
+  /**
+   * Join/Predicate Ordering rules can enable this cache by calling `join_graph_statistics_cache.emplace()`. The docs of
+   * JoinGraphStatisticsCache explain what this cache does.
```

Comment:
This comment does not tell me anything.

Reply:
Now, that's an unspecific change request. ... 

Anyway, I removed the comments here as they are essentially duplicates of the doc of `AbstractCardinalityEstimator::guarantee_join_graph()/guarantee_bottom_up_construction()` - to which the doc now simply points.